rom/dev/src/flow/cold_reset/fmc_alias.rs

```diff
@@ -358,6 +382,8 @@ impl FmcAliasLayer {
         env: &RomEnv,
         input: &DiceInput,
         output: &DiceOutput,
+        not_before: &[u8; FmcAliasCertTbsParams::NOT_BEFORE_LEN],
```

Comment:
Can we use a single constant?

Reply:
These are two constants which are auto generated. One constant is for NOT_BEFORE and another one is for NOT_AFTER. So I am using the respective constants.